### ISSUE TITLE
PostgreSQL is not starting in LCOW mode

### ISSUE THREAD

**Issue by FeodorFitsner** [appveyor/ci#3267]
Original report: https://help.appveyor.com/discussions/questions/43754-visual-studio-2019-docker-networking-issue
Workaround: https://github.com/docker/for-win/issues/445#issuecomment-547194464

```
icacls C:\ProgramData\docker\volumes /grant *S-1-5-83-0:"(OI)(CI)F" /T
```

**Comment by casz** [comment]
Even after applying the workaround and using a docker managed volume, I still have issues with spinning on gitlab on appveyor.

I'll try and provide a repro

**Comment by casz** [comment]
In my repro I cannot find the exact reason for gitlab container to report unhealthy.

https://ci.appveyor.com/project/casz/appveyor-gitlab-container-repro
https://github.com/casz/appveyor-gitlab-container-repro

Perhaps you have more luck testing / consuming the log on a VM directly

**Comment by FeodorFitsner** [comment]
Provided build log is the successful one - I don't see any errors there. Could you please elaborate what should I look for?

**Comment by casz** [comment]
The build log is not accurate as the container still reported unhealthy.

locally the container spins up correctly.

**Comment by FeodorFitsner** [comment]
Are you running the container locally in LCOW mode or on MobyLinuxVM?

**Comment by casz** [comment]
Whatever the default is 🤔

**Comment by casz** [comment]
Locally I use MobyLinuxVM as I run on Windows 10.
Where LCOW mode was removed.

**Comment by FeodorFitsner** [comment]
I mean your Docker Desktop is in Linux mode when you run GitLab container, right?

**Comment by casz** [comment]
Yes running docker desktop in linux mode when I launch GitLab container.

We are running the same test on GitHub actions on their ubuntu image with zero issues. Only reason we aren't using GitHub's windows image yet is they have not enabled LCOW nor provide a way to switch to Linux mode.

**Comment by FeodorFitsner** [comment]
On AppVeyor you can switch Docker to Linux mode with `Switch-DockerLinux` cmdlet. Give it a try. LCOW mode is really unstable sometimes, especially for complex things.
